rust/src/lib/nm/nm_dbus/connection/route.rs

```diff
@@ -26,18 +30,32 @@ pub struct NmIpRoute {
 impl TryFrom<DbusDictionary> for NmIpRoute {
     type Error = NmError;
     fn try_from(mut v: DbusDictionary) -> Result<Self, Self::Error> {
+        let dest = _from_map!(v, "dest", String::try_from)?;
+        let default_next_hop = dest.as_deref().and_then(|dest| {
+            match dest.parse::<std::net::IpAddr>() {
+                Ok(std::net::IpAddr::V4(_)) => Some(IPV4_EMPTY_NEXT_HOP_ADDR),
```

Comment:
If I do that, every time an `NmIpRoute` is created we will have a String allocation, but in most cases it won't be used. As I have done, here I return an `Option<&str>` and only call `to_string()` if it's needed, which is not often because almost all routes have next-hop values.

Reply:
Ah, no, I didn't understood your comment. With your change I would use the value of `dest` as "empty next-hop" value, but I have to use IPV6_EMPTY_NEXT_HOP_ADDR / IPV6_EMPTY_NEXT_HOP_ADDR instead.